lang/ilias_hr.lang

```diff
@@ -1960,7 +1960,6 @@ auth#:#login_page_switch_rte#:#Use Rich-text editor
 auth#:#lti_consumer_inactive#:#LTI tool consumer is disabled.
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#Online Now
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]

```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.